docs/guides/allmaps.md

```diff
@@ -28,5 +28,17 @@ Anyone can contribute to our collection of georeferenced map overlays.
 > - Try to distribute GCPs somewhat evenly throughout the image.
 7. After a sufficient number of points have been placed, click **Results**, then click **View current image.** Check to make sure the overlay looks reasonably accurate. If needed, use the back button and click **Georeference** to return to the editor and make changes. If the results aren't satisfactory, *delete all GCPs* so the scanned map is no longer georeferenced.
 
+## View a Georeferenced Map
 
+1. To view the overlay without leaving the resource page of a georeferenced scanned map, click the **Georeferenced Map** tab next to the **Item Viewer** tab beneath the title of the page. 
+2. To open the Allmaps Viewer in a new tab, click **View this georeferenced item** beneath the **Georeferencing** heading in the right sidebar of the resource page. 
+
+## Edit a Previously Georeferenced Map
+
```

Comment:
This won't always work, as there are two types of IIIF links: API and Manifest. Right now, our Geoportal only shows the button for APIs, which not all of our records have.  I will make an issue to have the Manifests show up as copyable links as well.